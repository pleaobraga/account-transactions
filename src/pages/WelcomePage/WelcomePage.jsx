import React from 'react'
import Card from '../../components/Card'
import './WelcomePage.scss'

const WelcomePage = () => {
  return (
    <main className="page page-welcome">
      <Card title="Welcome" text="This is a Rect Scaffold made by pleaobraga" />
    </main>
  )
}

export default WelcomePage
