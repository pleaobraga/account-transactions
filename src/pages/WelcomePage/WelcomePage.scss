.page {
  &-welcome {
    align-items: flex-start;
  }
}
