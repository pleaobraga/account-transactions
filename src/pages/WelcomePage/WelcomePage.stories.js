import React from 'react'
import { storiesOf } from '@storybook/react'
import WelcomePage from './WelcomePage'

export default storiesOf('Pages | WelcomePage', module).add(
  'default',
  () => <WelcomePage />,
  { info: { inline: true, header: false } }
)
