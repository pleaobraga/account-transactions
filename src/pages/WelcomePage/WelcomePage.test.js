import React from 'react'
import { shallow } from 'enzyme'
import WelcomePage from './WelcomePage'

describe('welcomePage', () => {
  const welcomePage = shallow(<WelcomePage />)

  it('render proprely', () => {
    expect(welcomePage).toMatchSnapshot()
  })
})
