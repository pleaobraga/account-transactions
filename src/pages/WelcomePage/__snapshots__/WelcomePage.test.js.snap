// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`welcomePage render proprely 1`] = `
<main
  className="page page-welcome"
>
  <Card
    text="This is a Rect Scaffold made by pleaobraga"
    title="Welcome"
  />
</main>
`;
